src/frontend/magnify/src/components/design-system/Formik/SubmitButton/FormikSubmitButton.tsx

```diff
@@ -1,58 +1,74 @@
-        defaultMessage: 'No changes detected',
-    },
-    formIsInvalid: {
-        id: 'components.designSystem.formik.formikSubmitButton.formIsInvalid',
-        description: 'Messages shown on hover when form is invalid ',
-        defaultMessage: 'Invalid form',
-    },
+  isNotDirty: {
+    id: 'components.designSystem.formik.formikSubmitButton.isNotDirty',
+    description: 'Messages shown on hover when form is not dirty ',
+    defaultMessage: 'No changes detected',
+  },
+  formIsInvalid: {
+    id: 'components.designSystem.formik.formikSubmitButton.formIsInvalid',
+    description: 'Messages shown on hover when form is invalid ',
+    defaultMessage: 'Invalid form',
+  },
 });
 
 interface FormikSubmitButtonProps extends ButtonExtendedProps {
-    overrideSubmit?: (values: unknown) => void;
+  overrideSubmit?: (values: unknown) => void;
 }
 
-export const FormikSubmitButton: FunctionComponent<FormikSubmitButtonProps> = ({...props}) => {
-    const formik = useFormikContext();
-    const intl = useIntl();
+export const FormikSubmitButton: FunctionComponent<FormikSubmitButtonProps> = ({ ...props }) => {
+  const formik = useFormikContext();
+  const intl = useIntl();
 
-    const onSubmit = (): void => {
-        if (props.overrideSubmit != null) {
-            props.overrideSubmit(formik.values);
-        } else {
-            formik.handleSubmit();
-        }
+  const onSubmit = (): void => {
+    if (props.overrideSubmit != null) {
+      props.overrideSubmit(formik.values);
+    } else {
+      formik.handleSubmit();
     }
 
-    const getTooltipMessage = (): string => {
-        if (!formik.dirty) {
-            return intl.formatMessage(formikSubmitButtonMessages.isNotDirty)
-        } else if (!formik.isValid) {
-            return intl.formatMessage(formikSubmitButtonMessages.formIsInvalid)
-        }
-        return '';
-    }
+    /**
+     * To simulate end of request.
+     * Move the logic back to the paren the parent component once the connection to the backend is made
+     * (in the onSubmit of the Formik form)
+     */
+    setTimeout(() => {
+      formik.setSubmitting(false);
+    }, 1000);
+  };
 
-    return <>
-        {(!formik.dirty || !formik.isValid) && <Tip content={getTooltipMessage()}>
-                <Box>
-                    <Button  {...props}
-                             primary
-                             disabled={true}/>
-                </Box>
-        </Tip>}
-        {formik.dirty && formik.isValid &&  <Button  {...props}
-                                                     primary
-                                                     onClick={onSubmit}/>}
-    </>
+  const getTooltipMessage = (): string => {
+    if (!formik.dirty) {
+      return intl.formatMessage(formikSubmitButtonMessages.isNotDirty);
+    } else if (!formik.isValid) {
+      return intl.formatMessage(formikSubmitButtonMessages.formIsInvalid);
+    }
+    return '';
+  };
```

Comment:
From my point of view, this method should return undefined not an empty string.
```suggestion
const getTooltipMessage = (): Maybe<string> => {
    if (!formik.dirty) {
      return intl.formatMessage(formikSubmitButtonMessages.isNotDirty);
    } else if (!formik.isValid) {
      return intl.formatMessage(formikSubmitButtonMessages.formIsInvalid);
    }
  };
```